src/modules/client/pages/ni/config/CustomersConfig.vue

```diff
@@ -600,6 +600,7 @@ export default {
       this.refreshServices(),
       this.refreshThirdPartyPayers(),
     ]);
+    this.documents = this.company.customersConfig.templates || {};
```

Comment:
Est-ce qu'on a besoin de ce documents ou est-ce qu'on peut directement passer par this.company.customersCOnfig.templates ?